src/System.Net.WebSockets.Client/dir.props

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\dir.props" />
```

Comment:
You need to add System.Net.WebSockets.Client to the packages.builds file in order to get the package built.

Reply:
What/where is the 'packages.builds' file?  

I thought the package builds on it own by doing the 'msbuild' in the System.Net.WebSocket.Client/pkg folder.

Reply:
You want to include it here so that it is built: https://github.com/dotnet/corefx/blob/release/1.1.0/src/packages.builds#L28

Reply:
Ok.  I pushed a commit to add that to the 'packages.build' file.